README.md

```diff
@@ -103,6 +107,29 @@ npx hardhat deploy --network localhost
 
 This makes the BOSON test token deployed on Rinkeby (0xEDa08eF1c6ff51Ca7Fd681295797102c1B84606c) and the official DAI token deployed on Rinkeby (0x6A9865aDE2B6207dAAC49f8bCba9705dEB0B0e6D) available to your local hardhat chain.
 
+
+#### Special deployments
+
+Calling `npx hardhat deploy` will deploy the protocol contracts. Beside that we provide addtional utility script for the following deployment cases:
+
+- Deploy all protocol contracts and ERC1155NonTransferable (equivalent to v1.0 deployment script)
+
+  ```
+  npx hardhat deploy-with-erc1155
+  ```
+- Deploy only ERC1155NonTransferable and set the metadata uri defined in `.env` file
+  ```
+  npx hardhat deploy-erc1155-only
+  ```
+- Deploy a set of mock token contracts ERC20, ERC721, ERC1155 and ERC1155NonTransferable which can be used to test conditional commit
+  ```
+  npx hardhat deploy-mocks
+  ```
+- Deploy a Gate contract on a provided network
```

Comment:
The gate deployment script won't work properly if the .env file isn't filled out properly. I suggest to change this section to the following markup:

- Deploy a Gate contract on a provided network
* First run either the deploy-with-erc1155 or deploy task to deploy the Boson Protocol contracts locally, as described above
* Then run deploy-mocks, as described above
* Provide values for the folowing properties in your .env file
  ```
  BOSON_ROUTER_ADDRESS=0000000000000000000000000000000000000000
  CONDITIONAL_TOKEN_ADDRESS=0000000000000000000000000000000000000000
  #Must be 0 (FUNGIBLE_TOKEN - ERC20), 1 (NONFUNGIBLE_TOKEN - ERC721), or 2 (MULTI_TOKEN - ERC1155)
  CONDITIONAL_TOKEN_TYPE=x

  ```
  The Boson Router address and conditional token address can be found in the corresponding file in the `addresses` directory. For local deployments, Boson Router will be in a filed called `1.json` (deploy), or `1-legacy.json` (deploy-with-erc1155). The mock conditional token addresses can be found in `1-mocks.json`

  Then run
  ```
  npx hardhat deploy-gate
  ```
Some of the markup formatting might have been lost when pasting it here.